services/fdb/server/conf_test.go

```diff
@@ -86,6 +87,16 @@ cluster_file = /etc/foundatindb/fdb.cluster`)
 	err = f1.Close()
 	testutil.FatalOnErr("can't close tmpfile", err, t)
 
+	// assign a special user, group and permission for this file
+	originUid, originGid, originMod := 1000, 1000, int(0775)
```

Comment:
You do not know if the userID/GroupID 1000 will exists in the environment you run. GH action would be running in a special environment so there is no guarantee. You can either create new user and chown/grp to it or list existing user and pick a non-root one.

Reply:
And the issue that unit test didn't get passed was because below statement requires root privilege so it threw operation not permitted. And when I ran unit tests in my devVM, they all passed because I was a root user. I found it was hard to grant this operation root like adding sudo in os/exec.Command() which requires password.
`os.Chown(name, originUid, originGid);`
My thought is to mock the chown operation(refer to: https://github.com/Snowflake-Labs/sansshell/blob/main/services/localfile/server/localfile_test.go#L540 ).

It might be the problem that the uid and gid may not be exist, but since I am going to mock the chown operation, not sure whether I still need to pick a real uid and gid.

Reply:
I guess setting it to 1000 is fine. You dont really have to have a user with that ID, it will just not show you a nice username. So it should be fine